atomix/cluster/src/main/java/io/atomix/raft/roles/PassiveRole.java

```diff
@@ -609,6 +609,10 @@ protected void appendEntries(
       raft.notifyCommitListeners(commitIndex);
     }
 
+    // Make sure all entries are flushed before ack
+    // because writer's buffer is isolated from readers
```

Comment:
Just a side note, most screens are probably big enough to put this on a single line ;)

Also, it's not because the writer's buffer is isolated from readers, but because for us to truly acknowledge the append to the leader, we need to ensure the write is truly persisted, not just buffered in memory and eventually flushed.

Reply:
>Just a side note, most screens are probably big enough to put this on a single line ;)

That's just like your opinion, man 😎

Reply:
> Also, it's not because the writer's buffer is isolated from readers, but because for us to truly acknowledge the append to the leader, we need to ensure the write is truly persisted, not just buffered in memory and eventually flushed.

It makes sense to explain this better. I'll change the comment ;)